### ISSUE TITLE
Search for few items

### ISSUE THREAD

**Issue by sivaletton** [vlourme/vintedpy#13]
Hi, after I start the bot, it sends a couple of results and then it stops, and it doesn't send anything anymore, do you know how I can fix it?

**Comment by vlourme** [comment]
Hello, do you have any error in the console?

Also, do you use multiple Vinted bot at the same time? Or have you reduced the `CHECK_INTERVAL` environment? These two things might cause your machine to be temporarily banned from Vinted servers. This bot does not uses proxy actually.

**Comment by sivaletton** [comment]
No, none of that, I don't have any errors and I haven't changed anything, I think it's the same problem as the #14  , Because it also returns me vinted.com where there are fewer results
